camtrapml/dataset.py

```diff
@@ -15,7 +15,7 @@ class ImageDataset:
     name: str
     path: Path
 
-    def __init__(self, name: str, path: Path):
+    def __init__(self,path: Path, name = None):
```

Comment:
Thanks for the contribution! It would be great to keep the type definition for `name` of `str`; maybe you could add that back?

Reply:
Apologies, yes; however, it sets the type only to be `None` for the name argument, rather than providing type hints for either `str` or `None`. Thus the typing should be specified with:

```python
from typing import Union

# ...

def __init__(self,path: Path, name : Union[None, str] = None):

# ...
```

Reply:
changed

Reply:
Fab. Cheers!